### PR TITLE
BETA: Register queaxtra.is-a.dev

### PR DESCRIPTION
**Pull request by Queaxtra**
Added `queaxtra.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/queaxtra.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Queaxtra",
+        "email": "fatihbgr@duck.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=queaxtra.is-a.dev,99c8c889ab11cf641590"
```
Comment (Stef-00012) [inline domains/queaxtra.json:7]: Vercel is not supported

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/queaxtra.json (hunk 1 above)
Comment (Queaxtra) [reply domains/queaxtra.json:7]: Can you please make the record value "CNAME" and "https://queaxtra-me.pages.dev/"? I migrated the site to Cloudflare Pages.

Comment (Stef-00012) [reply domains/queaxtra.json:7]: You can do it yourself, just edit the file in your fork

**Comment by Stef-00012** [comment]
Seems good now

**Pull request closed without merging**